site/en/user_guide.md

```diff
@@ -414,9 +433,9 @@ Type 'ocrd-' followed by `TAB` to get a list of all available processors.
 To get further information about one processor type
 
 ```sh
-[name_of_selected_processor] -J
+[name_of_selected_processor] -h
```

Comment:
I wouldn't leave out JSON completely. Perhaps we could mention it after help?

Reply:
`-h` includes everything from `-J` as well (but puts it in context)